website/variables.tf

```diff
@@ -1,7 +1,11 @@
 variable "prefix" {
-    description = "A prefix label, like wwtdev"
+  description = "A prefix label, like wwtdev"
 }
 
 variable "location" {
-    description = "The location where resources will be created"
+  description = "The location where resources will be created"
+}
+
+variable "resource_group_name" {
+  description = "Name of resource group"
```

Comment:
I don't think this new (well, re-added) variable gets used anywhere? The resource group name is derived from `var.prefix`.

Reply:
oh yeah - didn't realize I had committed that. Fixed